pillar/top.sls

```diff
@@ -346,3 +346,6 @@ base:
     - fluentd.ocw_db
   'P@roles:ocw-(cms|db|origin|mirror) and G@ocw-environment:production':
     - datadog
+  'G@roles:ocw-next-build and G@ocw-environment:production':
```

Comment:
The ocw-environment isn't going to be set as this isn't getting deployed to the OCW environment. Also, we need to have a match statement for both QA and production environments to load the separate pillar files. Alternatively, we should combine the two pillar files into a single one using a mapping object to handle the differences.

Reply:
Can you check my thinking here?
I'm seeing that the salt minion's environment grain(?) appears to be set [here](https://github.com/mitodl/ol-infrastructure/blob/17e4ebb49185683e227e26d96b3b00091b88e3c7/src/ol_infrastructure/applications/ocw_build/__main__.py#L153). If I understand this correctly, it's going to have an environment grain (`environment`?) that equals `applications-production`, based on what I see [here](https://github.com/mitodl/ol-infrastructure/blob/17e4ebb49185683e227e26d96b3b00091b88e3c7/src/ol_infrastructure/applications/ocw_build/__main__.py#L38). Is that right?

Therefore, I'm thinking this line in top.sls should say `G@roles:ocw-build and G@environment:applications-production`.

Reply:
Yes, that is correct

Reply:
Thanks!

Changed in f4a6b85a